index.html

```diff
@@ -31,16 +31,106 @@ <h1>Melanie Cascante</h1>
     </section>
 
     <div class="box">
-        <a href="#">
-            <i class="Start_collab"></i>
+        <a href="#" class="button">
+            <i class="Start_collab"></i>Start Collaboration</a>
     </div>
 
-    <div class="start_box">Start collaboration</div>
-
-
     <section class="geometrybottom">
         <img src="images/image_geometry_2.png">
     </section>
+
+    <div>
+        <h1>Projects</h1>  
+    </div>
+
+    <div class="rectangle21">
+        <img src="images/Rectangle 21.png">
+    </div>
+
+    <section class="works">
+        <div class="item1">Hello Microverse!</div>
+        <div class="item2">Html</div>
+        <div class="item3">Css</div>  
+        <div class="item4">JavaScript</div>
+        <div class="item5">
+            <a href="https://github.com/Maedea9/Hello-World3" class="button">
+                <i class="see-project"></i>See this project &#8594;</a>
+        </div>
+    </section>
+
+    <div class="rectangle21v">
+        <img src="images/Rectangle 21v.png">
+    </div>
+
+    <section class="works1">
+      
+        <div class="item6">Portfolio Mobile First</div>
+      <ul>
+        <li><div class="item7">Html</div></li>
+        <li><div class="item8">Css</div></li> 
+        <li><div class="item9">JavaScript</div></li>
+      </ul>
+        <div class="item10">
+            <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+                <i class="see-project"></i>See this project &#8594;</a>
+        </div>
+    </section>
+    
+    <section class="aboutme">
+        <h1>About</h1>
```

Comment:
- [ ] Kindly change this `About` header tag from `h1` to another `hx` tag as you already have an `h1` tag in the project and semantically it is advisable to have only one `h1` tag in a webpage for easy readability and accessibility. 👍